src/main/java/frc/robot/RobotContainer.java

```diff
@@ -99,6 +98,22 @@ private void controllerBindings() {
       } else {
         tankSubsystem.setTankDrivePowers(-joystickLeft.getY(), -joystickRight.getY());
       }
+
+      // "A" button controls open and closing of the claw
+      if (mechControlXbox.getAButtonReleased()) {
+        clawSubsystem.clawIsOpen = !clawSubsystem.clawIsOpen;
+      }
+
+      // "B" button controls lifting of the claw
+      if (mechControlXbox.getBButtonPressed()) {
+        clawSubsystem.clawIsLifted = !clawSubsystem.clawIsLifted;
+      }
+
+      // "X" button: closes and lifts claw at the same time
+      if (mechControlXbox.getXButtonPressed()) {
+        clawSubsystem.closeAndLiftCommand();

```

Comment:
a good idea; this probably won't give the claw enough time to close tho

Reply:
created a new command that lifts *after* the closing has completed